src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/ContinueSetupForSingleReservation.cshtml

```diff
@@ -0,0 +1,37 @@
+﻿@model SFA.DAS.EmployerAccounts.Web.ViewModels.AccountDashboardViewModel
```

Comment:
I don't think so because the view isn't called directly the model is extracted from the return of another method,the Row1Panel1 partial child action method.